containers/tefca-viewer/src/app/query/test/page.tsx

```diff
@@ -32,13 +32,9 @@ const Query: React.FC = () => {
             setUseCaseQueryResponse={setUseCaseQueryResponse}
             setOriginalRequest={setOriginalRequest}
             userJourney="test"
-            setUseCase={function (useCase: USE_CASES): void {
-              throw new Error("Function not implemented.");
-            }}
             setQueryType={function (queryType: string): void {
               throw new Error("Function not implemented.");
```

Comment:
```suggestion
              throw new Error("Function not implemented.");
```
I don't think we want to throw this error. Possibly a leftover?

Reply:
Shockingly not a leftover but actually a deliberate part of the built CustomizeQuery component. In practice, this function does absolutely nothing though; the demo user journey just uses `setQueryType` to assign state so that the right `queryName` can be looked up in `CustomizeQuery`. Since the customization flow isn't available in the `test` workflow, there's actually no effect at all; you can still hit `Fill Fields` and submit patients and get results back. That being said, because it does absolutely nothing, happy to replace with a `void` do nothing function altogether.